### ISSUE TITLE
curl: (52) Empty reply from server

### ISSUE THREAD

**Issue by snowmanirl** [biggis-project/biggis-hdfs#7]
When I run docker-compose up -d successfully,I encounter this warning:npm notice created a lockfile as package-lock.json. You should commit this file.
I'm not sure if it's related to the problem when I run the demo Upload hamlet.txt to HDFS :
curl -u hdfs:password \
     -F 'file=@data/hamlet.txt' \
     -X POST http://localhost:3000/api/v1/upload/files?hdfspath=/demo/hamlet.txt
The result shows curl: (52) Empty reply from server.
Before I execute it,I could access http://localhost:3000/ page. After that,I couldn't and the hdfs-api service in docker stops. I'm not familiar with node.js. Anyone can help me with this?

**Comment by vsimko** [comment]
yes, `package-lock.json` should definitely be under version control.
Even better, use `yarn` which has a similar file `yarn.lock`

**Comment by tosen1990** [comment]
@vsimko I try to rebuild the image by adding package-lock.json and all the steps process okay.But I still got this kind of error.  curl: (52) Empty reply from server

**Comment by vsimko** [comment]
I cannot answer your question in this case. But @wipatrick might.
